src/main/java/org/jongo/RxJongo.java

```diff
@@ -0,0 +1,46 @@
+package org.jongo;
```

Comment:
Since I cannot remember the exact technical conditions under which placing this in `org.jongo` is bad, it does not seem like that can be a factor.  I would probably look for something like this in the `org.jongo` or `org.jongo.util`, before I looked in `org.jongo.transformer`.  Perhaps it is best to leave it in `org.jongo`, until there are a few of them and they must be moved to a separate package.